modules/subscriptions/views.php

```diff
@@ -150,6 +163,22 @@ static function render_widget_status_messages( $instance ) {
 							__( 'Manage your email preferences.', 'jetpack' )
 						); ?></p>
 					<?php break;
+				case 'many_pending_subs':
+					?>
+					<p class="error">
+						<?php
+						printf(
+							wp_kses(
+								/* translators: this message is displayed after submitting an email in subscription widgets  */
```

Comment:
When providing context, it's often most helpful to specify what the placeholders stand for. For example here you could mention that the link leads to a Subscription management page.